src/routes/GroupExplorer/GroupSearch/GroupSearch.test.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import { unmountComponentAtNode } from 'react-dom'
+import { graphql } from 'msw'
+import { setupServer } from 'msw/node'
+import { history } from '../../../router'
```

Comment:
Use absolute `src` rooted imports. In this case you'd need to do `... from 'router/index`. Below `... from 'store/models`...